test/integration/contract-aci.js

```diff
@@ -304,6 +304,45 @@ describe('Contract instance', function () {
     result.callerId.should.be.equal(onAccount)
   })
 
+  describe('Transaction options', async () => {
+    it('deploys with no options', async () => {
+      const testContract = await aeSdk.getContractInstance({
+        source: testContractSource, filesystem
+      })
+      const deployInfo = await testContract.deploy(['test', 1, 'hahahaha'])
+      expect(deployInfo.address).to.satisfy(b => b.startsWith('ct_'))
+      expect(deployInfo.txData.tx.gas).to.be.above(0)
+      expect(deployInfo.txData.tx.amount).to.be.equal(0)
+      expect(deployInfo.txData.gasUsed).to.be.above(0)
+      expect(testContract.bytecode).to.satisfy(b => b.startsWith('cb_'))
+      testContractAddress = deployInfo.address
+    })
+
+    it('deploys with options', async () => {
```

Comment:
I would update the "deploys" test instead: https://github.com/aeternity/aepp-sdk-js/blob/2cc3b9137c1d1c88dcce5eb6875188d198690ce8/test/integration/contract-aci.js#L184-L192

Reply:
Sure